### PR TITLE
Controllers spec

### PR DESCRIPTION
**Pull request by momotsow**
In this pull request I :

- Add RSpec as a gem to the project.
- Create a Request spec file for users and post controllers.
- Check for each action :
- If the response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+RSpec.describe UsersController, type: :request do
+  before(:all) do
+    @first_user = User.create(name: 'happen', photo: 'photo', bio: 'test')
+  end
+
+  describe 'GET index' do
+    it 'renders index' do
+      get :index
+      expect(response.status).to eq(200)
+      expect(response).to render_template('index')
```
Comment (tufoinnkuo10) [inline spec/requests/users_spec.rb:12]: - [x] Great job so far.  please kindly include tests that ensures the response body includes the correct placeholder text.
Example : `expect(response.body).to include("Welcome to user index page")`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @momotsow 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ RSpec gem for testing

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by EricMbouwe — APPROVED** [review]
Hi @momotsow, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (momotsow) [reply spec/requests/users_spec.rb:12]: Hi @tufoinnkuo10 

Thank you for your review, This has been resolved

**Pull request merged**